lib/redis/time_series/version.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
+
 class Redis
   class TimeSeries
-    VERSION = '0.8.1'
+    VERSION = "0.8.8"
```

Comment:
This will definitely be an 0.9.x (maybe 1.0, if we can check all the boxes there)